config/archetype/pom.xml

```diff
@@ -21,7 +21,7 @@
   <parent>
     <groupId>org.kuali.rice</groupId>
     <artifactId>rice-config</artifactId>
-    <version>ua-2.5-release1</version>
+    <version>2.5.19-uar11-SNAPSHOT</version>
```

Comment:
In the tags for this repo there is a "ua-release10".  I wasn't sure if there was a release cadence the team was following where "11" would be the next release.  I used "uar11" as a shorter way to represent "UA Release 11".